### PR TITLE
query for country facet data, then merge into the countryList and cou…

### PR DESCRIPTION
**Pull request by mcstover**
…ntries data. create dynamic lend links for coutnries with available loans.

### CODE REVIEW

**Review by emuvente — APPROVED** [review]
👍

**Comment by mcstover** [comment]
I've moved the apollo query into the client. Once it resolves it will update the existing countryList and countries data.

Feel free to pull down and refine this as needed.

File: src/components/15Years/15YearsHeader.vue [hunk 1]
```diff
@@ -302,14 +304,41 @@ export default {
...
-		preFetch: true,
-		result({ data }) {
-			console.log('Fetched data from graphql~!', data);
+		fetchLiveCountryData() {
+			const query = this.apollo.query({
+				query: countryQuery,
+			});
+			query.then(({ data }) => {
+				// flatten country object and pass to list builder
+				const countries = data.lend?.countryFacets.map(obj => obj.country);
```
Comment (rally-adam) [inline src/components/15Years/15YearsHeader.vue:313]: All these changes scan and make sense to me, but this line throws an error for me in Storybook; `data` is coming in as undefined. Do I need to be running a separate process for graphql?

Comment (mcstover) [reply src/components/15Years/15YearsHeader.vue:313]: ahh yes my apologies... I didn't mock out any data for storybook...

Comment (mcstover) [reply src/components/15Years/15YearsHeader.vue:313]: I'll work on adding that in a few

**Review by emuvente — APPROVED** [review]

**Review by rally-adam — APPROVED** [review]

**Pull request merged**